src/components/pages/AdminAddCourses/AdminCourseFeed.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import '../../../styles/index.less';
+
+export default function AdminCourseFeed() {
+  const [courses, setCourses] = useState([]);
+  const token = JSON.parse(localStorage.getItem('okta-token-storage'));
+  const config = {
+    headers: { Authorization: `Bearer ${token.idToken.value}` },
+  };
```

Comment:
So it will be better to use the axiosWithAuth that has been merged to the front-end. It will help clean up the code. Great job setting up the bearer token verification for testing.